### PR TITLE
auth sets in

### PR DESCRIPTION
**Pull request by emmanuelle-c**

### CODE REVIEW

**Review by YannLahellec — APPROVED** [review]

File: server/app/services/auth.js [hunk 1]
```diff
@@ -16,4 +16,36 @@ const isAuth = async (req, res, next) => {
   }
 };
 
-module.exports = {isAuth};
+const isAdmin = async (req, res, next) => {
+  try {
+    if (req.auth.role === "admin") {
+      next();
+    }
```
Comment (Alexandre78R) [inline server/app/services/auth.js:23]: add else response error 403 not perission

File: server/app/services/auth.js [hunk 2]
```diff
@@ -16,4 +16,36 @@ const isAuth = async (req, res, next) => {
...
+    console.error(error);
+    res.sendStatus(403);
+  }
+};
+
+const isCrew = async (req, res, next) => {
+  try {
+    if (req.auth.role === "crew") {
+      next();
+    }
```
Comment (Alexandre78R) [inline server/app/services/auth.js:37]: add else response error 401 not perission

File: server/app/services/auth.js [hunk 3]
```diff
@@ -16,4 +16,36 @@ const isAuth = async (req, res, next) => {
...
+  } catch (error) {
+    console.error(error);
+    res.sendStatus(403);
+  }
+};
+const isClient = async (req, res, next) => {
+  try {
+    if (req.auth.role === "client") {
+      next();
+    }
```
Comment (Alexandre78R) [inline server/app/services/auth.js:50]: add else response error 403 not perission

**Review by Alexandre78R — CHANGES_REQUESTED** [review]

**Review by Alexandre78R — APPROVED** [review]
Good !

**Pull request merged**